src/app/views/marketplace/marketplace.html

```diff
@@ -19,6 +19,16 @@ <h2 class="section-title">{{ 'mno_enterprise.templates.dashboard.marketplace.ind
         <div class="col-sm-4">
           <input type="text" class="form-control" placeholder="{{ 'mno_enterprise.templates.dashboard.marketplace.index.search' | translate }}" ng-model="vm.searchTerm" ng-change="vm.selectedCategory = ''"/>
         </div>
+        <div class="col-sm-4 text-center">
+          <button ng-click="compare = !compare" ng-hide="compare" class="btn btn-primary">
```

Comment:
Please move the logic in the ng-click expression into a method in the controller. If variables are being set in ng attributes anywhere else in your PR can you also use controller methods instead.